src/microcosm.js

```diff
@@ -260,6 +260,22 @@ class Microcosm extends Emitter {
     // Remove all listeners
     this.removeAllListeners()
   }
+
+  parallel(actions) {
+    return this.append('GROUP').link(actions)
+  }
+
+  sequence(prepared) {
+    let payload
+
+    let sequence = () => function * () {
+      for (var i = 0; i < prepared.length; i++) {
+        payload = yield prepared[i](payload)
+      }
+    }
+
+    return this.push(sequence)
+  }
```

Comment:
ahh, that i'm not sure about. the reason this is so clean is because the work for walking through actions sequentially is already done in `coroutine::processGenerator`. something like